src/elf/mod.rs

```diff
@@ -202,22 +202,45 @@ if_sylvan! {
         pub fn is_object_file(&self) -> bool {
             self.header.e_type == header::ET_REL
         }
+
+        pub fn parse_elf_hdr(bytes: &'a [u8]) -> error::Result<Header> {
```

Comment:
why is this a pub fn (or a function at all actually?)

Reply:
Can we rename this to parse_header then since elf is redundant a bit since user would write `Elf::parse_elf_hdr`